tests/dummy/app/styles/app.scss

```diff
@@ -104,7 +98,7 @@ a {
     h1,
     p {
       @include font-smoothing;
-      color: $c-text-color;
+      color: _e3dn('text-color');
```

Comment:
I definitely think that makes sense. We don't want to impose styles on anything but the things the addon touches. All other styles should be applied in the dummy app directly.